docs/useful-libraries/react/README.md

```diff
@@ -2,25 +2,25 @@
 
 ## General
 
-* [Reach UI](https://reacttraining.com/reach-ui/)
+* [Reach UI](https://reacttraining.com/reach-ui/) - Accessible foundation for React apps
 
 ## Hooks
 
-* [react-use-measure](https://github.com/react-spring/react-use-measure)
-* [react-intersection-observer](https://github.com/thebuilder/react-intersection-observer)
-* [react-use](https://github.com/streamich/react-use)
-* [react-adaptive-hooks](https://github.com/GoogleChromeLabs/react-adaptive-hooks)
-* [use-persisted-state](https://github.com/donavon/use-persisted-state)
-* [use-query-string](https://github.com/trevorblades/use-query-string)
-* [promise-hook](https://github.com/aiven715/promise-hook)
-* [react-hotkeys-hook](https://github.com/JohannesKlauss/react-hotkeys-hook)
+* [react-use-measure](https://github.com/react-spring/react-use-measure) - Reactive boundry measurments
+* [react-intersection-observer](https://github.com/thebuilder/react-intersection-observer) - Check if element is in the viewport
+* [react-use](https://github.com/streamich/react-use) - Large collection of essential hooks
+* [react-adaptive-hooks](https://github.com/GoogleChromeLabs/react-adaptive-hooks) - Hooks and utilities for adaptive loading
```

Comment:
Maybe we could list a couple of the hooks from these collections to give an idea of the sorts of things available